[microTVM] Fix warnings on Zephyr tests

gromero opened the pull request:
Fix the following warning message on Zephyr tests:

DeprecationWarning: Please use input parameter mod (tvm.IRModule)
instead of deprecated parameter mod (tvm.relay.function.Function)

Signd-off-by: Gustavo Romero <gustavo.romero@linaro.org>

Thanks for contributing to TVM!   Please refer to guideline https://tvm.apache.org/docs/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

Mousius commented on tests/micro/zephyr/test_zephyr_aot.py:
```diff
@@ -233,6 +233,7 @@ def test_qemu_make_fail(temp_dir, platform, west_cmd, tvm_debug):
     xx = relay.multiply(x, x)
     z = relay.add(xx, relay.const(np.ones(shape=shape, dtype=dtype)))
     func = relay.Function([x], z)
+    func = tvm.IRModule.from_expr(func)
```
Given this isn't the function anymore, maybe we should go with `ir_mod` or `mod` ?

gromero commented:
@Mousius @areusch Thanks for the quick review! I've just pushed https://github.com/apache/tvm/commit/00b49dc40d94a75dfb551785c52dff4634dba0ef addressing the comment from Chris.

Mousius reviewed (approved):
Thanks for the update @gromero, LGTM!

gromero commented:
@areusch @leandron I'm wondering if you could please collect it? :)

leandron reviewed (approved):
LGTM, thanks @gromero and @Mousius!
